Flaky TestArgComplete.test_compare_with_compgen

Example failure:

```
__________________ TestArgComplete.test_compare_with_compgen ___________________
[gw3] linux2 -- Python 2.7.6 /home/travis/build/pytest-dev/pytest/.tox/py27-nobyte/bin/python
self = <test_argcomplete.TestArgComplete object at 0x7f6da5d3a7d0>
    @pytest.mark.skipif("sys.platform in ('win32', 'darwin')")
    def test_compare_with_compgen(self):
        from _pytest._argcomplete import FastFilesCompleter
    
        ffc = FastFilesCompleter()
        fc = FilesCompleter()
        for x in ["/", "/d", "/data", "qqq", ""]:
>           assert equal_with_bash(x, ffc, fc, out=sys.stdout)
E           AssertionError: assert False
E            +  where False = equal_with_bash('', <_pytest._argcomplete.FastFilesCompleter object at 0x7f6da5d881d0>, <test_argcomplete.FilesCompleter object at 0x7f6da5d88150>, out=<_pytest.capture.EncodedFile object at 0x7f6da6f9cbd0>)
E            +    where <_pytest.capture.EncodedFile object at 0x7f6da6f9cbd0> = sys.stdout
testing/test_argcomplete.py:96: AssertionError
```

Always on py27.

Seen on https://github.com/pytest-dev/pytest/pull/4161 and #4153.

--- a/tox.ini
+++ b/tox.ini
@@ -21,7 +21,7 @@ commands =
     {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof
     coverage: coverage combine
     coverage: coverage report
-passenv = USER USERNAME COVERAGE_*
+passenv = USER USERNAME COVERAGE_* TRAVIS
 setenv =
     # configuration if a user runs tox with a "coverage" factor, for example "tox -e py36-coverage"
     coverage: _PYTEST_TOX_COVERAGE_RUN=coverage run -m

